fluxx.game.php

```diff
@@ -64,12 +64,19 @@ public function __construct()
       "drawnCards" => 20,
       "playedCards" => 21,
       "lastGoalBeforeDoubleAgenda" => 30,
+      "hasDoubleAgenda" => 31,
```

Comment:
It's a detail, but we should have the same variable name for all. Either we go with "has", or with "active", but we should have the same for all :-)